packager/getFlowTypeCheckMiddleware.js

```diff
@@ -30,7 +29,17 @@ function getFlowTypeCheckMiddleware(options) {
       }
       return next();
     }
-    exec('command -v flow >/dev/null 2>&1', function(error, stdout) {
+
+    // If the flow-bin package is installed then use that. Otherwise use
+    // whatever is in the path
+    var flowbin;
+    try {
+      flowbin = require('flow-bin');
+    } catch (ex) {
+      flowbin = 'flow';
+    }
+
+    exec('command -v '+flowbin+' >/dev/null 2>&1', function(error, stdout) {
```

Comment:
can you escape `flowbin` variable. if the folder name has some bad characters you can execute anything here